README.md

```diff
@@ -14,6 +14,8 @@ Sample Unreal Game using Unified SDK features provided by <https://github.com/im
 
 ## Cloning this Repo
 
+Before cloning this repo, ensure you have [git lfs](https://git-lfs.com/) installed.
```

Comment:
See https://github.com/immutable/unreal-immutable-sdk/pull/26/files

Reply:
Saw that. I didn't want to mention the file types that Git LFS tracks because that's the job of .gitattributes - if it changes, then the readme is out of date.